metrics/lib/common.bash

```diff
@@ -42,6 +42,47 @@ function save_results(){
 	bash $LIB_DIR/send_results.sh -n "$1" -a "$2" -r "$3" -u "$4"
 }
 
+# This function checks existence of commands.
+# They can be received standalone standalone or as an array, e.g.
+#
+# cmds=(“cmd1” “cmd2”)
+# check_cmds "${cmd[@]}"
+function check_cmds()
+{
+	req_tools="$@"
```

Comment:
Declaring it local fixes one of the 3 things that is wrong.

The best test case is to add another directory to you PATH, and put a command in it with a space in the name, e.g. "do it", and then see what happens when you call

     check_cmds "do it"